src/platform/plugins/shared/saved_objects_management/public/management_section/objects_table/components/table.tsx

```diff
@@ -369,26 +372,100 @@ export class Table extends PureComponent<TableProps, TableState> {
       );
     }
 
-    const button = (
-      <EuiButton
-        iconType="arrowDown"
-        iconSide="right"
-        onClick={this.toggleExportPopoverVisibility}
-        isDisabled={selectedSavedObjects.length === 0}
-      >
-        <FormattedMessage
-          id="savedObjectsManagement.objectsTable.table.exportPopoverButtonLabel"
-          defaultMessage="Export"
-        />
-      </EuiButton>
-    );
-
     const activeActionContents = this.state.activeAction?.render() ?? null;
     const exceededResultCount = totalItemCount > MAX_PAGINATED_ITEM;
 
+    const hasMLJobs = selectedSavedObjects.some(({ type }) => type === 'ml-job');
+    const hasMLModels = selectedSavedObjects.some(({ type }) => type === 'ml-trained-model');
+    const hasMlObjects = hasMLJobs || hasMLModels;
+
     const anySelected = selectedSavedObjects.length > 0;
     const allHidden =
       anySelected && selectedSavedObjects.every(({ meta: { hiddenType } }) => hiddenType);
+
+    const deleteTooltip = () => {
+      if (hasMLJobs && hasMLModels) {
+        return i18n.translate(
+          'savedObjectsManagement.objectsTable.table.deleteMLJobsAndModelsDisabledTooltip',
+          {
+            defaultMessage:
+              'Navigate to the Machine Learning management pages to delete machine learning jobs and trained models.',
```

Comment:
Can we use `management.getInAppUrl` in the SO definition to take the users to the ML management pages instead?

Reply:
Yes that's the plan. This is dependent on another PR that we will have merge soon. I'll add a todo to make notes.